$MirOS: ports/net/wget/patches/patch-src_utils_c,v 1.1.7.1 2005/03/18 15:49:22 tg Exp $
--- src/utils.c.orig	Mon May 30 13:31:02 2005
+++ src/utils.c	Wed Jun 29 13:58:13 2005
@@ -302,8 +302,8 @@ datetime_str (time_t *tm)
       return output;
     }
   ptm = localtime (&secs);
-  sprintf (output, "%04d-%02d-%02d %02d:%02d:%02d",
-	   ptm->tm_year + 1900, ptm->tm_mon + 1, ptm->tm_mday,
+  sprintf (output, "%04ld-%02d-%02d %02d:%02d:%02d",
+	   (long)ptm->tm_year + 1900, ptm->tm_mon + 1, ptm->tm_mday,
 	   ptm->tm_hour, ptm->tm_min, ptm->tm_sec);
   return output;
 }
